translations.json

```diff
@@ -1009,7 +1009,7 @@
         },
         "TabsESLabel": {
             "en": "Event Source",
-            "zh-CN": "事件源"
+            "zh-CN": "基本设置"
```

Comment:
As far as I can tell this translates to "basic settings" which would be weird since this appears on Cactbot's settings tab as well.
With this change you'd end up with two "basic settings" tabs if Cactbot is enabled: One would be labeled "basic settings" and the other would be labeled "Cactbot | basic settings".
It's not wrong but it seems a bit confusing to me.

Reply:
Some Chinese users don't know the meaning about "EventSource"(事件源),it makes them confuse."Why the config page names EventSource?"They ask me as that lot of times.Actually I don't want to modify it like this,but I can't think of a better way.

Reply:
And the EventSource of Cactbot texts "CactbotEventSource" has not translate into Chinese.It shows like this.Some Chinese users don't know it is the config page. 
![image](https://user-images.githubusercontent.com/44015352/74096013-14594a00-4b34-11ea-9056-e3beff601c6e.png)